### ISSUE TITLE
OrmLite Alias same name as a different field no longer supported

### ISSUE THREAD

**Issue by georgehemmings** [ServiceStack/Issues#469]
I've just upgrade from OrmLite 4.0.56 to 4.0.60 and have found a scenario that is no longer supported.

I'm working against a legacy db that has some incorrect column names. One table `AddressAudit` has a PK named `AddressId`  and a FK column named `AddressLinkId`. I previously used alias to map them to *better* names:

AddressId > Id
AddressLinkId > AddressId

However since upgrading to 4.0.60 the `Id` field is no longer set. It looks like the field name is now taking priority over the `Alias`. Was this intended?

```csharp
public class AddressAudit
{
    [PrimaryKey, AutoIncrement]
    [Alias("AddressId")]
    public int Id { get; set; }

    [Alias("AddressLinkId")]
    public int AddressId { get; set; }
}

// 'addressAudit.Id' is not set
var addressAudit = Db.Single<AddressAudit>(audit => audit.AddressId == debtor.AddressId);
```

**Comment by mythz** [comment]
This issue was due to having an alias that had the same name as another field which should now be fixed in this commit: https://github.com/ServiceStack/ServiceStack.OrmLite/commit/2aa4b483df9ff3e5019c19ea7baed88185587008

This change is available from v4.0.61 that's now [available on MyGet](https://github.com/ServiceStack/ServiceStack/wiki/MyGet).

**Comment by georgehemmings** [comment]
Thanks!
